src/server/status/collectors/get_ops_stats_collector.js

```diff
@@ -44,6 +44,7 @@ export function getOpsStatsCollector(server, kbnServer) {
         kibana: getKibanaInfoForStats(server, kbnServer),
         ...kbnServer.metrics // latest metrics captured from the ops event listener in src/server/status/index
       };
-    }
+    },
+    internalIgnore: true, // Ignore this one from internal uploader. A different stats collector is used there.
```

Comment:
I'm wondering if this can be more verbose to avoid ambiguity - maybe `ignoreForInternalUploader`?